lib/src/services/tmdb-service.dart

```diff
@@ -1,25 +1,40 @@
 import 'dart:async';
 import 'dart:convert';
 import 'package:http/http.dart';
+import '../models/content-base.dart';
 import '../models/country.dart';
 import '../models/genre.dart';
-import '../models/movie-base.dart';
 import '../models/movie.dart';
 import '../models/paged-result.dart';
+import '../models/tv-show.dart';
+import '../settings/append-settings.dart';
 import '../settings/asset-resolver.dart';
 import '../settings/configuration.dart';
-import '../settings/movie-append-settings.dart';
-import '../settings/movie-search-settings.dart';
+import '../settings/search-settings.dart';
 import '../settings/quality-settings.dart';
 
+part 'common-service.dart';
 part 'configuration-service.dart';
 part 'movie-service.dart';
 part 'resilient-service.dart';
+part 'tv-service.dart';
 
-class TmdbService extends Object
-    with MovieService, ConfigurationService, ResilientService {
+class TmdbService extends Object with ConfigurationService {
   final String _apiKey;
 
+  MovieService _movie;
+  TvService _tv;
+
+  MovieService get movie {
```

Comment:
Two things here:
1. If someone modifies the configuration after the MovieService/TVService is created, the configuration update will not be propagated to the services (edge case really, but probably has to be covered)
2. I am wondering if a check for configuration not being null should be done before creating the MovieService/TVService as the services will be unusable without proper config being initialized before.